docs/about/index.md

```diff
@@ -0,0 +1,86 @@
+.NET Documentation
+==================
+
+Welcome to .NET! You can build any kind of application you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next application today, on Windows, OS X and Linux. This documentation shows you how.
+
+Languages
+---------
+
+.NET is multi-language as a design point. It supports multiple languages and enables interop between them. C#, F# and Visual Basic are the most popular. C# and F# are both multi-paradigm, with C# being primarily object-oriented and F# primarily functional. Visual Basic is the most productive and easiest to learn. The following C# example demonstrates the most basic console app.
+
+
+```csharp
+using System;
+
+class Program
+    {
+        static void Main(string[] args)
+        {
+        	Console.WriteLine("Hello World");
+        }
+    }
+```
+
+Try .NET
+--------
+
+Try out .NET Core. It is the quickest and easiest way to try out .NET, on Windows, macOS or Linux. You can be running .NET Core on your machine in 5 minutes. Try it now or watch it below. TODO: Add link here.
+
+TODO: Animation or video of those 5 mins. 
+
+Use .NET for Any App
+--------------------
+
+You can build any app with .NET. There are many highly capable and popular .NET solutions, app types and libraries that you can use, provided by various companies and individuals.
+
+* Web
+* Mobile
+* Cloud
+* Desktop
+* Games
+* IoT
+* Open Source
+
+TODO: Add detail pages for all of these and then link to those.
+
+Learning about .NET
+-------------------
+
+You can learn about .NET, the best ways to use it and what kind of apps you can build with it. 
+
+.NET
+----
+
+.NET is a general purpose development platform. It is used for many kinds of apps and workloads by using the many [.NET Products](products.md) that are available. It has several key features that make it easy and efficient to build great applications, including automatic memory management and modern programming languages. .NET enables a high-level programming environment with many convenience features, while providing low-level access to native memory and APIs.
```

Comment:
The doc earlier talks about there being many kinds of apps; I'd be inclined to use this section to briefly explain what you mean by ".NET" i.e., what is this thing that applies to all the ".NET products" and that I might want to learn about independently from the "products"

"easy" is generally seen as a marketing term -- it's never really easy to make real-world applications

per my TOC comment, I would send the learn .NET link to concepts under fundamentals rather than under About.